src/components/BaseComponents/BaseMarker.tsx

```diff
@@ -14,20 +14,21 @@ type BaseMarkerProps = {
     handleClick?: any;
     stationId: number;
     isDay: boolean;
```

Comment:
Do we still need this isDay property here?
Seems it's now redundant, but maybe I missed sth